webrtc.html

```diff
@@ -4712,9 +4712,15 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <dt><dfn><code>codecs</code></dfn> of type <span class=
             "idlMemberType">sequence&lt;<a>RTCRtpCodecParameters</a>&gt;</span></dt>
             <dd>
-              <p>A sequence containing the codecs that an
-              <a><code>RTCRtpSender</code></a> will choose from in order to
-              send media.</p>
+            <p>
+              A sequence containing the codecs (including RTX/RED/FEC) that
+              an <code><a>RTCRtpSender</a></code> will choose from in order
+              to send media. Corresponding to each media codec where retransmission
+              via rtx is enabled, there will be an entry in <code>codecs[]</code>
+              with a <var>mimeType</var> value indicating retransmission via rtx,
+              but with a distinct <var>sdpFmtpLine</var> attribute (providing an
+              "apt" parameter).
```

Comment:
Instead of relying on the APT in the sdpFmtpLine, why don't we just have an associatedPayloadType field on the RTCRtpCodecParameters?  That would make it much easier for applications to read it.